app/config/config.dev.ts

```diff
@@ -7,5 +7,6 @@ import { Platform } from "react-native"
  * https://reactnative.dev/docs/security#storing-sensitive-info
  */
 export default {
+  // TODO: sort out why this is necessary for android
```

Comment:
https://stackoverflow.com/a/43277765

Basically in package.json there is already a script setup for creating the bridge for android. We probably need to modify it to work with localhost:3000

`    "adb": "adb reverse tcp:9090 tcp:9090 && adb reverse tcp:3000 tcp:3000 && adb reverse tcp:9001 tcp:9001 && adb reverse tcp:8081 tcp:8081",
`